src/main/java/com/phonebookservice/controller/AbstractController.java

```diff
@@ -0,0 +1,33 @@
+package main.java.com.phonebookservice.controller;
+
+import main.java.com.phonebookservice.model.Model;
+import main.java.com.phonebookservice.server.DBIf;
+
+public abstract class AbstractController {
+    private DBIf databse = null;
```

Comment:
There are two convension of interfaces 

no prefix or suffix for interfaces, ~Impl suffix for implementations
~I prefix for interfaces, but no prefix or suffix for implementations.


which means IDB or DB and all inherited classes as Impl suffix (FileAdapterImpl)